go.mod

```diff
@@ -22,7 +22,7 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.4.0
 	github.com/stretchr/testify v1.5.1
-	golang.org/x/tools v0.0.0-20200403190813-44a64ad78b9b
+	golang.org/x/tools v0.0.0-20200916150407-587cf2330ce8
 	gomodules.xyz/jsonpatch/v3 v3.0.1
 	helm.sh/helm/v3 v3.2.4
 	k8s.io/api v0.18.6
```

Comment:
Is not it missing the make file target to create the bindata ? would not it be part of this PR? How we will add the next ones?

Reply:
I am not sure if we need to add a Makefile target for this, as this is only an internal optimization which I doubt will be used by developers too. But I do have a script which would do this? Do we need to commit it too?

Reply:
@varshaprasad96 the makefile target that I am speaking about is into the SDk repo and not in the projects built by it. For us/maintainers be able to maintain this implementation and built the bindatas for the  OLM versions when required.